tests/diff/blob.c

```diff
@@ -520,19 +520,19 @@ void test_diff_blob__can_compare_a_binary_blob_and_a_text_blob(void)
  * +++ b/a0f7217
  * @@ -1,6 +1,6 @@
  *  Here is some stuff at the start
- * 
+ *
```

Comment:
Spurious whitespace change ? The build was red, but I haven't yet looked at the build log.